server/src/test/java/io/whitefox/api/server/DeltaUtils.java

```diff
@@ -0,0 +1,16 @@
+package io.whitefox.api.server;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class DeltaUtils {
+
+  private static final Path deltaTablesRoot = Paths.get(".")
+      .toAbsolutePath()
```

Comment:
Don't we already have this somewhere else? If you refactoring it out, can you please refactor also the other usages?